src/components/events-page/events-feed.jsx

```diff
@@ -20,12 +20,23 @@ let EventsFeed = ({
     <BlockHeader size="h1">Events</BlockHeader>
 
     {(filteredEventsTags.length > 0 || !selectedTags.length) && (
-      <TagList
-        label="Events tags"
-        tags={filteredEventsTags.length > 0 ? filteredEventsTags : selectedTags}
-        selectedTags={selectedTags}
-        onTagClick={toggleTag}
-      />
+      <TagListCutter
```

Comment:
To tell the truth this composition looks pretty complex.
It would be much simpler if the logic related to expanding/collapsing tags just live in Tags component.